projects/1.21/assets/simple-conveyor-belts/belts/lang/zh_cn.json

```
@@ -0,0 +1,23 @@
+{
+  "message.belts.reset": "传送带已重置！",
+  "message.belts.started": "起点已存储！",
+  "message.belts.belt_created": "传送带已铺设！",
+  "message.belts.midpoint_added": "途经点已存储！",
+  "message.belts.midpoint_duplicate": "此途经点已被存储过。",
+  "message.belts.chute_used": "溜槽被占用！",
+  "message.belts.filter_set": "物品过滤器已设置",
+  "message.belts.filter_reset": "物品过滤器已重置",
+  "itemgroup.belts.items": "Simple Belts",

```

```suggestion
  "itemgroup.belts.items": "便捷传送带",
```
仅建议，避免以后被直接翻译成简易传送带（感觉简易意思不是很准确）